core/lib/globals_shared.h

```diff
@@ -1873,6 +1878,7 @@ typedef union _dr_simd_t {
 #    define PRE_SIMD_PADDING                                       \
         0 /**< Bytes of padding before xmm/ymm dr_mcontext_t slots \
            */
+#    define MCXT_NUM_OPMASK_SLOTS 0 /**< No simd masks */
```

Comment:
Please check whether this is the version that makes it into the html: since AARCHXX is listed under PREDEFINED in API.doxy I'm afraid it is, which will result in "No simd masks" in our official docs, which is not what we want.  Please use the same general description "Number of..." as in the Doxygen comment below: avoid this "No simd masks" to avoid having incorrect docs in the resulting generated html.

Reply:
Ok, yes it is. I sync'd the comment, but added "if architecture supports".